hack/log-dump/log-dump.sh

```diff
@@ -109,6 +109,20 @@ function dump-systemd-log() {
   done
 }
 
+# Convert json-based logs to slightly more human-readable logs
+function post-dump-log() {
+  local -r log_files=( $(find "${ARTIFACTS}" -name "*.log") )
+  for log_file in "${log_files[@]}"; do
```

Comment:
the real logs may also contain those characters, could we parse the results with jq? we may need to check whether jq is installed on the prow containers

Reply:
jq is installed by default - https://github.com/kubernetes/test-infra/blob/master/images/bootstrap/Dockerfile#L36.